control/control.yaml

```diff
@@ -107,3 +109,23 @@ components:
           type: string
           enum: [start, stop]
 
+    Control.BgpControlState:
+      description: >-
+        Control BGP Protocol state to handle special case.
+        Because generic protocol control handle within FlowTransmitState
+      type: object
+      required: [state]
+      properties:
+        names:
+          description: >-
+            The names of BGP object to control.
+          type: array
+          items:
+            type: string
+            x-constraint:
+            - '/components/schemas/Device.Bgpv4'
```

Comment:
constraint is the full path of the name of the bgpv4 object -> /components/schemas/Device.Bgpv4/properties/name

Reply:
Thanks!